src/main/proto/TokenDissociate.proto

```diff
@@ -33,7 +33,7 @@ import "BasicTypes.proto";
  If any of the provided tokens is not found, the transaction will resolve to INVALID_TOKEN_REF.
  If any of the provided tokens has been deleted, the transaction will resolve to TOKEN_WAS_DELETED.
  If an association between the provided account and any of the tokens does not exist, the transaction will resolve to TOKEN_NOT_ASSOCIATED_TO_ACCOUNT.
- If the provided account has a nonzero balance with any of the provided tokens, the transaction will resolve to TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES.
+ If the provided account has a nonzero balance with any of the provided tokens, the transaction will resolve to TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES, even if the token has already expired.
```

Comment:
Actually this is not quite right
- Both fungibles & non-fungibles -> If `token is not deleted && not expired` && `user has != 0 balance` result is `TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES
- For Fungibles -> If `token is expired`, user can disassociate even if he has token balance
- For Non fungibles -> If `token is expired`, user **cannot** disassociate if he has token balance and `TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES` is returned